engine/src/main/java/org/hibernate/search/engine/search/SearchResult.java

```diff
@@ -17,6 +17,12 @@
 
 	List<T> getHits();
 
-	// TODO aggregations, facets, ...
+	// TODO implement => T getAggregations();
```

Comment:
Please remove these changes. None of them makes sense in the `SearchResult` class from the engine. You can get rid of the code responsible for computing these values, too; it will be re-implemented differently.